src/components/Blockquote.tsx

```diff
@@ -17,91 +17,37 @@ interface Link {
-            <h3 className={`font-weight-normal ${header ? 'text-2xl' : 'text-3xl'}`}>&ldquo;{quote}&rdquo;</h3>
-            {author && <figcaption className="pt-3 text-muted text-center">&mdash; {author}</figcaption>}
-        </blockquote>
-        {logo && (
-            <div className="d-flex justify-content-center">
-                {logo.href ? (
-                    <a href={logo.href} className="btn">
-                        <img src={logo.src} width="110px" alt={logo.alt} />
-                    </a>
-                ) : (
-                    <img src={logo.src} width="110px" alt={logo.alt} />
-                )}
-            </div>
-        )}
-        {link &&
-            (link.href.includes('http') ? (
-                <a href={link.href} target="_blank" rel="nofollow noreferrer">
-                    {link.text}
-                    <ArrowRightIcon className="ml-1" />
-                </a>
-            ) : (
-                <Link href={link.href} passHref={true}>
-                    {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-                    <a className="d-flex justify-content-center mt-3">
-                        <p className="font-weight-bold">{link.text}</p>
-                        <ArrowRightIcon className="ml-1" />
-                    </a>
-                </Link>
-            ))}
-    </>
-)
-
-export const BlockquoteWithLogoTop: FunctionComponent<{
-    quote: string
-    author?: string | ReactFragment
-    logo?: Logo
-}> = ({ quote, author, logo }) => (
-    <>
-        {logo && (
-            <div className="d-flex justify-content-center">
-                {logo.href ? (
-                    <a href={logo.href} className="btn">
-                        <img src={logo.src} width="150px" alt={logo.alt} />
-                    </a>
-                ) : (
-                    <img src={logo.src} width="150px" alt={logo.alt} />
-                )}
-            </div>
-        )}
-        <blockquote className="p-3 rounded rounded-lg d-flex flex-column bg-transparent">
-            <h2 className="display-3 font-weight-bold">&ldquo;{quote}&rdquo;</h2>
-            {author && <figcaption className="pt-3 text-gray-5 text-center">&mdash; {author}</figcaption>}
-        </blockquote>
-    </>
-)
-
 export const Blockquote: FunctionComponent<{
     quote: string | ReactFragment
-    author?: string
+    author?: string | ReactFragment
     logo?: Logo
     link?: Link
     headline?: string
     largeText?: boolean
     center?: boolean
-}> = ({ quote, author, logo, link, headline, largeText = false, center = false }) => (
-    <>
-        <blockquote className={classNames('border-left border-3 border-vivid-violet', center && 'text-center')}>
-            {headline && <h4 className="font-weight-bold mb-4 px-4">{headline}</h4>}
+    border?: boolean
+}> = ({ quote, author, logo, link, headline, largeText = false, center = true, border = true }) => (
```

Comment:
Since our blockquotes are usually centered, I made centered truthy as default so we have to assign props less often to this component (Will be easier for blog writers, as well)